### ISSUE TITLE
DEL(point) should work

### ISSUE THREAD

**Issue by mdsumner** [hypertidy/anglr#115]
I.e. 

```R
DEL(sf::st_cast(silicate::minimal_mesh, "POINT"))
DEL(sf::st_cast(silicate::minimal_mesh, "MULTIPOINT"))
DEL0(sf::st_cast(silicate::minimal_mesh, "MULTIPOINT"))
DEL0(sf::st_cast(silicate::minimal_mesh, "POINT"))
```

there's no edges, but all the rest should work area, angle, etc.

**Comment by mdsumner** [comment]
In DEL.PATH0:

```R  #-------------------------------
  #if (silicate:::topology_type(x) == "1-space") {
    ## bail out with a point-triangulation
    ## FIXME: unimplemented
  #}
  #----------------
```

**Comment by mdsumner** [comment]
fixed in 17c9333635c77510e493d311605299d2925d9b25
